src/migrateContentServer.test.ts

```diff
@@ -232,4 +232,42 @@ describe("migrateContentServer", () => {
       undefined,
     );
   });
+
+  it("migrates the kpi custom titles when the original version contains a custom title with a tupleKey that is empty.", async () => {
+    const contentServer: ContentRecord = {
+      children: { ui: smallLegacyUIFolder, pivot: smallLegacyPivotFolder },
+      entry: {
+        owners: [],
+        readers: [],
+        isDirectory: true,
+        canRead: true,
+        canWrite: false,
+        lastEditor: "Freddie Mercury",
+        timestamp: 0xbeef,
+      },
+    };
+
+    await migrateContentServer({
+      contentServer,
+      servers,
+      fromVersion: "4.3",
+      toVersion: "5.1",
+      keysOfWidgetPluginsToRemove: [],
+      doesReportIncludeStacks: false,
+      shouldUpdateFiltersMdx: true,
+      behaviorOnError: "keep-original",
+    });
+
+    const migratedKpiContent = JSON.parse(
+      contentServer.children?.ui.children?.widgets.children?.content.children?.[
+        "kpi"
+      ].entry.content,
+    );
```

Comment:
```suggestion
    // The widget with id "kpi" contains a custom title referenced with an empty `tupleKey`.
    // Because this title doesn't represent any tuple, it is dropped.
    const kpiContentBeforeMigration = contentServer.children?.ui.children?.widgets.children?.content.children?.[
        "kpi"
      ].entry.content;
      
    const migratedKpiContent = JSON.parse(kpiContentBeforeMigration);
```